docs/csharp/language-reference/operators/and-operator.md

```diff
@@ -18,19 +18,21 @@ author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # &amp; Operator (C# Reference)
```

Comment:
You've made a great sweep of these articles. Can you update the ms.date metadata value to "04/03/2018"